.github/workflows/darwin.yml

```diff
@@ -2,59 +2,54 @@ name: macOS build for vnu
-    # - uses: actions/setup-python@v2
-    #   with:
-    #     python-version: '3.x'
-    # - name: Update Submodules
-    #   run: python ./checker.py update
-    # - name: Install Dependencies
-    #   run: python ./checker.py dldeps
-    # - name: Build
-    #   env:
-    #     TOMCAT_VERSION: 8.5.29
-    #   run: |
-    #     python ./checker.py build
-    #     python ./checker.py jar
-    #     python ./checker.py image
-    #     python ./checker.py war
-    #     java -jar ./build/dist/vnu.jar ./build/dist/index.html
-    #     java -jar ./build/dist/vnu.jar ./site/nu-about.html
-    #     curl -O https://archive.apache.org/dist/tomcat/tomcat-8/v$TOMCAT_VERSION/bin/apache-tomcat-$TOMCAT_VERSION.zip
-    #     unzip apache-tomcat-$TOMCAT_VERSION.zip
-    #     chmod a+x apache-tomcat-$TOMCAT_VERSION/bin/*.sh
-    #     jar xf ./build/dist/vnu.jar && file nu/validator/client/SimpleCommandLineValidator.class | grep "version 52.0"
-    #     jar xf ./build/dist-war/vnu.war && file WEB-INF/classes/nu/validator/servlet/VerifierServlet.class | grep "version 52.0"
-    #     apache-tomcat-$TOMCAT_VERSION/bin/catalina.sh start
-    #     cp ./build/dist-war/vnu.war apache-tomcat-$TOMCAT_VERSION/webapps/
-    #     sleep 15s; tail apache-tomcat-$TOMCAT_VERSION/logs/catalina.out || true
-    #     curl -sS 'http://localhost:8080/vnu/?out=gnu&doc=data:text/html;charset=utf-8,%3C%21doctype%20html%3E%3Cmeta%20charset%3Dutf-8%3E%3Ctitle%3ETest%3C%2Ftitle%3E' > RESULTS
-    #     test -z "$(if [ -t RESULTS ]; then cat RESULTS | grep -v '^$'; fi)"
-    #     if [ $? != 0 ]; then cat RESULTS; fi
-    #     python ./checker.py check
-    #     python ./checker.py test
-    # - uses: actions/upload-artifact@v2
-    #   if: matrix.java == 11
-    #   with:
-    #     # Currently creates a nested zip https://github.com/actions/upload-artifact/issues/3
-    #     name: vnu.osx.zip
-    #     path: "build/dist/vnu.osx.zip"
+      - uses: actions/checkout@v2
+      - name: Set up Java JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 16
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Update submodules
+        run: python ./checker.py update
+      - name: Install dependencies
+        run: python ./checker.py dldeps
+      - name: Build
+        uses: ./.github/actions/build
+      - name: Import GPG signing key
+        uses: crazy-max/ghaction-import-gpg@v3
+        if: github.ref == 'refs/heads/main'
+        with:
+          gpg-private-key: ${{ secrets.GPG_SIGNING_KEY }}
+          passphrase: ${{ secrets.GPG_SIGNING_KEY_PASSPHRASE }}
+          git-user-signingkey: true
+          git-commit-gpgsign: true
+      - name: Sign release artifacts
+        if: github.ref == 'refs/heads/main'
+        run: |
+          for file in build/dist/{*.zip,*.zip.md5,*.zip.sha1}; do gpg --yes -ab --sign $file; done || true
```

Comment:
Do the Zips need to be renamed to include the platform like `vnu.osx.zip` here before the signing?

Reply:
> Do the Zips need to be renamed to include the platform like `vnu.osx.zip` here before the signing?

No, I think they’ve already got the right names. Take a look at https://github.com/sideshowbarker/validator/releases (where I have been testing this)

Reply:
Yeah, looks like it has the platform specific parts https://github.com/sideshowbarker/validator/releases/tag/latest
Side comment: Does it make sense to sign the other hash files (MD5/SHA1)

Reply:
> Take a look at [sideshowbarker/validator/releases](https://github.com/sideshowbarker/validator/releases)

By the way, the reason I didn’t turn of the separate `jar` and `linux` tag releases yet is that there’s code in some other repos depending on those. So, after we merge this, I’m going to open PRs for updating the code in those other repos to use the `latest`-tag releases instead. And once that’s done, I’ll update the repo actions and releases settings to remove those two tags/releasee